### PR TITLE
[wct] Decrease possibility of child suites (iframes) being retained in memory

### PR DESCRIPTION
**Pull request by merlinnot**
Event listeners were not removed after an execution of a suite.

This is a step towards resolving #229.

### CODE REVIEW

File: packages/web-component-tester/browser/childrunner.ts [hunk 1]
```diff
@@ -10,6 +10,9 @@
  */
 import * as util from './util.js';
 
+const DOM_CONTENT_LOADED_EVENT_NAME = 'DOMContentLoaded';
+const IFRAME_ERROR_EVENT_NAME = 'error';
```
Comment (rictic) [inline packages/web-component-tester/browser/childrunner.ts]: Local style is `lowerCamelCase` for constants.

File: packages/web-component-tester/browser/childrunner.ts [hunk 2]
```diff
@@ -167,25 +194,34 @@ export default class ChildRunner {
...
-    setTimeout(function() {
-      this.iframe.parentNode.removeChild(this.iframe);
-      this.iframe = null;
-    }.bind(this), 1);
+    if (this.iframe) {
+      // Be safe and avoid potential browser crashes when logic attempts to
+      // interact with the removed iframe.
+      setTimeout(() => {
+        const { iframe } = this;
+        iframe.removeEventListener(
```
Comment (rictic) [inline packages/web-component-tester/browser/childrunner.ts]: For this sort of cleanup work, I prefer an array of callbacks to call as a private member on the class. That way the cleanup work can be registered at the same time and in the same place as the stuff that needs cleanup.

That would also let us avoid the repetitive file-level constants for event names

**Comment by merlinnot** [comment]
@rictic I've changed the approach a little, I hope you'll find it aligned with your concept.

**Comment by merlinnot** [comment]
@rictic a friendly ping :)

**Comment by merlinnot** [comment]
@rictic Travis throws `session not created exception: Chrome version must be >= 66.0.3359.0`.

**Comment by usergenic** [comment]
Thanks for PR @merlinnot - Integrating now.

**Pull request merged**